docs/podman-cp.1.md

```diff
@@ -54,6 +54,11 @@ You can also use : when specifying paths to a **SRC_PATH** or **DEST_PATH** on a
 If you use a : in a local machine path, you must be explicit with a relative or absolute path, for example:
 	`/path/to/file:name.txt` or `./file:name.txt`
 
+## OPTIONS
+
+**--extract**
+
+Extract the tar file into the destination directory. If the destination argument is not give, extract the tar file in root directory.
```

Comment:
suggest:

"Extract the tar file into the destination directory. If the destination option is not provided, extract the tar file into the root directory."

Reply:
oops, my bad, try 

"Extract the tar file into the destination directory. If the destination directory is not provided, extract the tar file into the root directory."